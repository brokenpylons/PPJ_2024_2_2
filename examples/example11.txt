var n = 10;
var a = 0;
var b = 1;
var c = 0;
for (var i = 1, n) {
    print(a);
    c = a + b;
    a = b;
    b = c;
}
