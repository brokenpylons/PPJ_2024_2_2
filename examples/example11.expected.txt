real("3") times("*") real("2") minus("-") real("1") 
